ios/NativeModules/TextileNode.m

```diff
@@ -237,30 +267,28 @@ - (void)_create:(NSString *)dataDir apiUrl:(NSString *)apiUrl logLevel:(NSString
     [config setCentralApiURL:apiUrl];
     [config setLogLevel:logLevel];
     [config setLogFiles:logFiles];
-    self.node = [[MobileMobile new] newNode:config messenger:[[Messenger alloc] init] error:error];
+    self.node = MobileNewNode(config, [[Messenger alloc] init], error);
   }
 }
 
-- (BOOL)_start:(NSError**)error {
-  BOOL startNodeSuccess = [self.node start:error];
-  return startNodeSuccess;
+- (void)_start:(NSError**)error {
```

Comment:
Since these `BOOL` return values are an artifact of gomobile and not used, no reason to return them from methods like this.